docs/syntax_and_semantics/c_bindings/enum.md

```diff
@@ -48,11 +48,11 @@ The type of an enum member is `Int32` by default, even if you specify a differen
 ```crystal
 lib X
   enum SomeEnum
-    A = 1_u32
+    A = 1_u32 # => Error: enum value must be an Int32
   end
 end
 
-X::SomeEnum # => 1_i32
+X::SomeEnum 
```

Comment:
**suggestion:** This line can be dropped entirely. Since the enum declaration already errors, this reference doesn't matter at all.

Reply:
This doesn't seem resoved to me. Or do you disagree with removing this line? (I'm happy if there's a good reason to keep it).

Reply:
thought i deleted this